Experimental/DedicatedGameServer/Assets/InputSystem_Actions.inputactions

```diff
@@ -0,0 +1,1057 @@
+{
```

Comment:
Is this Input asset being used? If so, could they be merged?
The player is using the asset inside the InputSystem folder.

Reply:
Nice catch, it seems this one was autogenerated with the update of the Input System package. I removed it and updated the project settings so that the default `inputactions` file one is the one you pointed out.

Thanks to a warning in the console, I also noticed that the existing InputSystem file doesn't have a configuration to handle "UI", so I'm following [the documentation ](https://docs.unity3d.com/Packages/com.unity.inputsystem@1.8/manual/UISupport.html#setting-up-ui-input)to implement it. (it's already implemented in the auto-generated file, so we can jsut copy-paste it).

This is now fixed!